packages/caliper-iroha/lib/iroha-connector.js

```diff
@@ -298,10 +291,10 @@ class IrohaConnector extends BlockchainConnector {
             ) {
                 throw new Error('Invalid Iroha::getContext arguments');
             }
-            let context = {};
+            this.context = {};
```

Comment:
seems unnecessary? Is instantly assigned in the next line

Reply:
True, that's an unintentional remnant 😅 Fixed.